src/client/app/chat/chat.component.ts

```diff
@@ -264,8 +267,16 @@ export class ChatComponent implements OnInit, AfterViewChecked {
     this.createForm();
     this.receiveMessageFromSocket();
     this.receiveUpdatedMessageFromSocket();
-    //this.safeUrl = this.domSanitizer.bypassSecurityTrustResourceUrl('this.selectedUser.appearUrl');
+    this.safeUrl = this.domSanitizer.bypassSecurityTrustResourceUrl('this.selectedUser.appearUrl');
```

Comment:
Wrong declaration.

Reply:
this code we are using for appearUrl, its not related to modal window

Reply:
We had a discussion before that you are not going to use domSanitizer in your implementation. Why are you still using it if it is not being implemented?

Reply:
No this one you have implemented for appearUrl .so I haven't removed, for modal window URL, i have removed.

Reply:
Oh... I am sorry. I thought you were trying to use this url in your implementation. How about the other changes I have mentioned?

Reply:
just removed from app.module.ts.That was the only one.